article.html

```diff
@@ -0,0 +1,607 @@
+                      </div>
+                  </div>
+                  <div class="title title-wrap">
+                      <div id="title-63091b8133" class="cmp-title">
+                        <h2 class="cmp-title__text">Section Title Ipsum Dolar</h2>
+                      </div>
+                  </div>
+                  <div class="text subtext-wrap quote-content">
+                      <div id="text-49e22e7256" class="cmp-text">
+                          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Mattis aliquam faucibus purus in. Venenatis lectus magna fringilla urna. Tempor orci eu lobortis elementum nibh. Risus nec feugiat in fermentum posuere urna nec tincidunt. Facilisis leo vel fringilla est. Lobortis ma purus in. Arcu odio ut sem nulla pharetra diam sit amet nisl. Enim sed faucibus turpis in eu mi bibendum neque. Sed elementum tempus egestas sed sed risus pretium quam.</p>
+                      </div>
+                  </div>
+                  <div class="image img-wrap">
+                    <div data-cmp-lazythreshold="0" data-cmp-src="/content/leaderimpact/lang/en2/_jcr_content/content-parsys/container_1715762034/container_1796181454/tabs/item_1669125273755/image_copy.coreimg{.width}.png/1669126312045/quote-img.png" data-asset="/content/dam/LeaderImpact/site-./assets/Rectangle_726.png" data-title="lady smiling" id="image-6bcf13ac84" class="cmp-image" itemscope="" itemtype="http://schema.org/ImageObject">
+                      <img src="./assets/Rectangle_709.png" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt." title="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt." >
+                      <span class="cmp-image__title" itemprop="caption">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt.</span>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <!-- ARTICLE SLIDER -->
+        <div class="container responsivegrid article-slider aem-GridColumn aem-GridColumn--default--12 layout-grid">
+          <div id="container-b0c5fbd9e2" class="cmp-container">
+            <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 ">
+              <!-- ARTICLE MAIN TITLE -->
+              <div class="responsivegrid aem-GridColumn title-btn-wrap aem-GridColumn--default--12 layout-grid">
```

Comment:
@Prystal254 The title shouldn't need this wrapping container.

Reply:
wrapping container removed from title